Core/Lexer/Extensions/CharExtensions.cs

```diff
@@ -410,6 +410,8 @@ internal static bool IsTypeSuffix(this char c)
             return false;
         }
 
-    #endregion
+        #endregion
+
+        public static char FileSeparator = '\u001c';
```

Comment:
This is a little bit tricky: the SchemaReader can only "talk to the outside world" using `char` values, but I want to report to the Tokenizer that the end of a single schema file was reached. So, I make the SchemaReader report an appropriately-named control character, which the tokenizer turns into an `EndOfFile` token.

The problem this solves is as follows (it was not introduced by `import` statements):

Suppose you `bebopc --files a.bop b.bop`, and `a.bop` contains `struct A { int32 x;` (with no closing brace), and `b.bop` contains `enum B { Foo = 0; }`.

Then instead of reporting "expected `}` but found `EndOfFile` in a.bop", the parser confusingly complains that it "expected `}` but found `enum` in b.bop".